src/msw/window.cpp

```diff
@@ -2812,8 +2909,14 @@ bool wxWindowMSW::MSWClickButtonIfPossible(wxButton* btn)
 #if wxUSE_BUTTON
     if ( btn && btn->IsEnabled() && btn->IsShownOnScreen() )
     {
+#ifndef __WXUNIVERSAL__
         btn->MSWCommand(BN_CLICKED, 0 /* unused */);
         return true;
+#endif // !__WXUNIVERSAL
+
+        wxCommandEvent event(wxEVT_BUTTON, btn->GetId());
+        event.SetEventObject(btn);
+        return btn->HandleWindowEvent(event);
```

Comment:
Shouldn't this be inside an `#else`? Right now it ought to provoke warnings about unreachable code.